storage.go

```diff
@@ -560,6 +616,27 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 		return nil, err
 	}
 
+	metadata := output.Metadata
+	for k, v := range metadata {
```

Comment:
How about use `metadata["x-amz-meta-bs-link-target"]` instead of range?

Reply:
OK